AirCasting/APICommunicator/APICommunication.swift

```diff
@@ -0,0 +1,31 @@
+// Created by Lunar on 01/07/2021.
+//
+
+import Foundation
+
+final class APICommunication {
+    func forgotPassword(login: String, completion: @escaping (Int) -> Void) {
+        let params = ["user": ["login": login]]
+
+        var request = URLRequest(url: URL(string: "http://aircasting.org/users/password.json")!)
+        request.httpMethod = "POST"
+        request.httpBody = try? JSONSerialization.data(withJSONObject: params, options: [])
+        request.addValue("application/json", forHTTPHeaderField: "Content-Type")
+        let session = URLSession.shared
+        let task = session.dataTask(with: request, completionHandler: { _, response, _ -> Void in
+
+            guard let httpResponse = response as? HTTPURLResponse else {
+                print("error: not a valid http response")
+                return
+            }
+            switch httpResponse.statusCode {
+            case 200, 201:
```

Comment:
also use the `HTTPResponseValidator` for response validation. Both of those protocols shall be injected into this class using initializer injection:
```Swfit
private let apiClient: APIClient
private let responseValidator: HTTPResponseValidator

init(apiClient: APIClient, responseValidator: HTTPResponseValidator) {
...
}
```